### PR TITLE
add o-typography Stories

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: components/o-typography/stories/typography-list.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		guidelines: {
+			notion: 'f82c8bd1ef1f4f649a61d17d5d49750c',
+		},
+		html: {},
+	},
+};
+
+const items = ['Apples', 'Bananas', 'Oranges', 'Pears', 'Tangelos'];
+const Story = args => <Body>
+    <List {...args} children={args.items.map(item => <ListItem key={nanoid()}>{item}</ListItem>)} />
```
Comment (chee) [inline components/o-typography/stories/typography-list.stories.tsx:26]: doesn't calling nanoid() like this mean there is a different key every time it rerenders?

Comment (JakeChampion) [reply components/o-typography/stories/typography-list.stories.tsx:26]: I think so!

Could we use the array index as the key?

Comment (chee) [reply components/o-typography/stories/typography-list.stories.tsx:26]: `key={item}` should be fine here

Comment (notlee) [reply components/o-typography/stories/typography-list.stories.tsx]: Oops, yes! I've removed that

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**